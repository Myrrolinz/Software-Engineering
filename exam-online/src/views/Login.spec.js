import { shallowMount } from '@vue/test-utils';
import Login from './Login';





describe('<Login/>', () => {
	const wrapper = shallowMount(Login);

	// 快照测试
	it('snapshot测试', () => {
		const wrapper2 = shallowMount(Login);
		expect(wrapper2.html()).toMatchSnapshot()
		wrapper2.destroy()
    })

	
  });

  /*
  it("slide correctly", () => {
		// 滑动 slide
		wrapper.find('slide-verification').trigger('check-result');
	  });
  		// 可以立即获取 msg 最新的值，不再需要 wrapper.vm.$nextTick();
		expect(wrapper.find('h1').text())
		  .toEqual('new message');
		  */
