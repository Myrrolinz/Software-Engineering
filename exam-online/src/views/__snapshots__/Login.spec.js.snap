// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> snapshot测试 1`] = `
"<div id=\\"login\\">
  <el-container-stub>
    <el-header-stub height=\\"60px\\">
      <h1 style=\\"color: rgb(255, 255, 255); margin-top: 50px;\\">英语四六级在线练习系统</h1>
    </el-header-stub>
    <el-main-stub>
      <div id=\\"login-from\\">
        <el-form-stub model=\\"[object Object]\\" rules=\\"[object Object]\\" labelsuffix=\\"\\" statusicon=\\"true\\" showmessage=\\"true\\" validateonrulechange=\\"true\\">
          <el-form-item-stub label=\\"学号\\" prop=\\"username\\" inlinemessage=\\"\\" showmessage=\\"true\\">
            <el-input-stub type=\\"text\\" autocomplete=\\"off\\" validateevent=\\"true\\"></el-input-stub>
          </el-form-item-stub>
          <el-form-item-stub label=\\"密码\\" prop=\\"password\\" inlinemessage=\\"\\" showmessage=\\"true\\">
            <el-input-stub type=\\"password\\" autocomplete=\\"off\\" validateevent=\\"true\\"></el-input-stub>
          </el-form-item-stub>
          <slide-verification-stub></slide-verification-stub> <br>
          <el-button-stub type=\\"primary\\">登录</el-button-stub>
          <div class=\\"text-foot\\">
            还没有账号?
            <router-link to=\\"/register\\" class=\\"register-link\\">
              注册
            </router-link>
          </div>
        </el-form-stub>
      </div>
    </el-main-stub>
  </el-container-stub>
</div>"
`;
